queue/queue.py

```diff
@@ -1,17 +1,29 @@
+from linked_list import LinkedList
```

Comment:
Apparently you can switch these (put this line at the bottom of the imports) and add `# noqa` to the end of the line to keep it in place.

Reply:
Mine still says `No LinkedList in module 'linked_list'`